### PR TITLE
Prepare for initial first v0.0.1 release

### PR DESCRIPTION
**Pull request by fatih**
* Update README.md with new instructions and better copy
* Add missing dependency to the plugin image
* Change image policy to the default one
* Fixed findmnt command output when the exit status is non zero

### CODE REVIEW

**Comment by jhernandezb** [comment]
Nice! I was just debugging this error 😄

File: README.md [hunk 1]
```diff
@@ -41,31 +42,43 @@ $ kubectl create -f ./secret.yml
...
 A new storage class will be created with the name `do-block-storage` which is
-responsible for dynamic provisioning. This is set to "default" for dynamic
+responsible for dynamic provisioning. This is set to **"default"** for dynamic
 provisioning. If you're using multiple storage classes you might want to remove
-the annotation from the `csi-storageclass.yaml` and re-deploy it.
+the annotation from the `csi-storageclass.yaml` and re-deploy it. This is
+based on the [recommended mechanism](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md#recommended-mechanism-for-deploying-csi-drivers-on-kubernetes) of deploying CSI drivers on Kubernetes
 
-This is based on the recommended mechanism of deploying CSI drivers on Kubernetes: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md#recommended-mechanism-for-deploying-csi-drivers-on-kubernetes
+*Note that the deployment proposal to Kubernetes is still work in progress and not all of the written
```
Comment (aybabtme) [inline README.md]: `is still _a_ work`

File: README.md [hunk 2]
```diff
@@ -41,31 +42,43 @@ $ kubectl create -f ./secret.yml
...
+features are implemented. When in doubt, open an issue or ask #sig-storage in [Kubernetes Slack](http://slack.k8s.io)*
 
-Note that the proposal is still work in progress and not all of the written
-features are implemented. When in doubt, ask #sig-storage in [Kubernetes Slack](http://slack.k8s.io)
+#### 3. Test and verify:
 
-3. Test and verify:
-
-Create a PersistenVolumeClaim. This makes sure a volume is created and provisioned on behalf of you:
+Create a PersistentVolumeClaim. This makes sure a volume is created and provisioned on behalf of you:
```
Comment (aybabtme) [inline README.md]: `on your behalf`

**Comment by aybabtme** [comment]
:shipit:

**Pull request merged**